Src/WitsmlExplorer.Frontend/components/Modals/AnalyzeGapModal.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import { NavigationAction } from "../../contexts/navigationAction";
+import { DisplayModalAction, HideModalAction } from "../../contexts/operationStateReducer";
+import LogObject from "../../models/logObject";
+import JobService, { JobType } from "../../services/jobService";
+import { WITSML_INDEX_TYPE_DATE_TIME, WITSML_INDEX_TYPE_MD } from "../Constants";
+import ModalDialog from "./ModalDialog";
+import { TextField } from "@equinor/eds-core-react";
+import OperationType from "../../contexts/operationType";
+import { ReportModal } from "./ReportModal";
+import WarningBar from "../WarningBar";
+
+export interface AnalyzeGapModalProps {
+  dispatchNavigation: (action: NavigationAction) => void;
+  dispatchOperation: (action: HideModalAction | DisplayModalAction) => void;
```

Comment:
No need to use prop-drilling for dispatchNavigation and dispatchOperation. Also, dispatchNavigation is not even used.

Reply:
fixed